.github/workflows/continuous-integration.yml

```
@@ -23,14 +22,15 @@ jobs:
           - "7.3"
           - "7.4"
           - "8.0"
+        dbal-version:
+          - "^2.12"
+          - "^3.0"
         dependencies:
           - "highest"
+          - "lowest"
         optional_dependencies:
           - true
           - false
-        include:
-          - php-version: "7.3"
-            dependencies: "lowest"
 
```

```suggestion
        include:
          - php-version: "7.3"
            dependencies: "lowest"
            dbal-version: "^2.12"
            optional_dependencies: false
          - php-version: "7.3"
            dependencies: "lowest"
            dbal-version: "^3.0"
            optional_dependencies: false
          - php-version: "7.3"
            dependencies: "lowest"
            dbal-version: "^2.12"
            optional_dependencies: true
          - php-version: "7.3"
            dependencies: "lowest"
            dbal-version: "^3.0"
            optional_dependencies: true
            
```

If we don't want to run lowest dependencies for all PHP version, we need to add these checks explicitly, as above.